tests/PfsFiberTrace.py

```diff
@@ -0,0 +1,202 @@
+            self.assertEqual(ftfc.order, ftfcNew.order)
+            self.assertEqual(ftfc.xLow, ftfcNew.xLow)
+            self.assertEqual(ftfc.xHigh, ftfcNew.xHigh)
+            self.assertEqual(ftfc.nPixCutLeft, ftfcNew.nPixCutLeft)
+            self.assertEqual(ftfc.nPixCutRight, ftfcNew.nPixCutRight)
+            self.assertEqual(ftfc.nRows, ftfcNew.nRows)
+
+            coeffs = ft.getTraceCoefficients()
+            coeffsNew = ftNew.getTraceCoefficients()
+            self.assertEqual(coeffs.shape[0], coeffsNew.shape[0])
+            for iCoeff in range(coeffs.shape[0]):
+                self.assertAlmostEqual(coeffs[iCoeff], coeffsNew[iCoeff], places=3)
+
+            xCenters = ft.getXCenters()
+            xCentersNew = ftNew.getXCenters()
+            self.assertEqual(xCenters.shape[0], xCentersNew.shape[0])
+            for iY in range(xCenters.shape[0]):
+                self.assertAlmostEqual(xCenters[iY], xCentersNew[iY], places=3)
+
+            profile = ft.getProfile().getArray()
+            profileNew = ftNew.getProfile().getArray()
+            self.assertEqual(profile.shape[0], profileNew.shape[0])
+            self.assertEqual(profile.shape[1], profileNew.shape[1])
+            for iX in range(profile.shape[0]):
+                for iY in range(profile.shape[1]):
+                    if np.abs(profile[iX,iY] - profileNew[iX,iY]) > 0.1:
+                        print 'FiberTrace ',iFt,': large difference detected: profile[',iX,',',iY,'] = ',profile[iX,iY],', profileNew[',iX,',',iY,'] = ',profileNew[iX,iY],'      xCenters[',iX,'] = ',xCenters[iX],', xCentersNew[',iX,'] = ',xCentersNew[iX]
+                    self.assertAlmostEqual(profile[iX,iY], profileNew[iX,iY], places=3)
+
+            image = ft.getImage().getArray()
+            imageNew = ftNew.getImage().getArray()
+            self.assertEqual(image.shape[0], imageNew.shape[0])
+            self.assertEqual(image.shape[1], imageNew.shape[1])
+            for iX in range(image.shape[0]):
+                for iY in range(image.shape[1]):
+                    if np.abs(image[iX,iY] - imageNew[iX,iY]) > 0.1:
+                        print 'FiberTrace ',iFt,': large difference detected: image[',iX,',',iY,'] = ',image[iX,iY],', imageNew[',iX,',',iY,'] = ',imageNew[iX,iY],'      xCenters[',iX,'] = ',xCenters[iX],', xCentersNew[',iX,'] = ',xCentersNew[iX]
+                    self.assertAlmostEqual(image[iX,iY], imageNew[iX,iY], places=3)
+
+            self.assertEqual(ft.getITrace(), ftNew.getITrace())
+            self.assertEqual(ft.getFiberTraceFunction().xCenter, ftNew.getFiberTraceFunction().xCenter)
+            self.assertEqual(ft.getFiberTraceFunction().yCenter, ftNew.getFiberTraceFunction().yCenter)
+            self.assertEqual(ft.getFiberTraceFunction().yLow, ftNew.getFiberTraceFunction().yLow)
+            self.assertEqual(ft.getFiberTraceFunction().yHigh, ftNew.getFiberTraceFunction().yHigh)
```

Comment:
Can checking the FiberTraceFunction be subsumed into `self.assertEqual(ft, ftNew)`?  If not, I suggest replacing most of this block with `self.assertEqual(ft.getFiberTraceFunction(), ftNew.getFiberTraceFunction())`.